server/plugin.go

```diff
@@ -116,15 +120,11 @@ func (p *Plugin) OnActivate() error {
 		p.configuration.SetDefaults()
 	}
 
-	if err = p.configuration.IsValid(); err != nil {
```

Comment:
Why did we remove this?

Reply:
Now we are calling `p.loadConfiguration` in the beginning of `OnActivate` and we already do the validation there.

Reply:
Also removing the `nil` check above, we don't need it.